UserLib.js

```diff
@@ -1,23 +0,0 @@
-$(function() {
```

Comment:
Why have you deleted this? We need this.

Reply:
I thought I'd created that by mistake in pinegrow😂
I can ad that back in